### PR TITLE
chore(bump): upgrade helm chart global dependencies

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR upgrades the helm chart dependencies to the latest version.

Cert manager cannot be upgraded to v1.16 because of this bug: https://github.com/cert-manager/cert-manager/issues/1829

The upgrade was already tested on a production k8s cluster by upgrading from the current version to the new one.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.


#### :radio_button: Update the status of your PR

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.




#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/openedx-k8s-harmony-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Review by bradenmacdonald — APPROVED** [review]
LGTM. Thanks @Ian2012! @gabor-boros did you want to take a look?

**Comment by Ian2012** [comment]
@bradenmacdonald I've not reviewed other helms charts because I don't have any use for them and I'm unsure how to test it. Good to know for a follow-up if anyone of you folks want to take a look.

File: charts/harmony-chart/Chart.yaml
Comment (jfavellar90) [inline charts/harmony-chart/Chart.yaml:17]: @Ian2012 we need to bump the chart version to a new one (line 8 of this file). I would suggest 0.8.0 as the new version

**Comment by bradenmacdonald** [comment]
If we can get one more +1 from either @gabor-boros or @jfavellar90 then I think this is good to merge.

**Review by jfavellar90 — APPROVED** [review]

**Comment by jfavellar90** [comment]
@gabor-boros Hi, we would like to get you involved in this conversation since we are not updating the following dependencies since we are not very familiar with testing them:

- Velero
- OpenFaas
- ElasticSearch and OpenSearch (I don't think it's worth it to update them since MeiliSearch is coming. Do you agree?)

Are you OK with this approach of merging this PR as it is and leaving the update of the missing dependencies for a follow-up task? Or do you prefer checking such dependencies in this PR to apply the update of all the dependencies?

**Comment by gabor-boros** [comment]
@jfavellar90 I won't be able to test Velero and OpenFaas today. However, I don't think I should block this PR to be merged.

So, let's merge this PR and I can update OpenFAAS and Velero separately later.

**Review by gabor-boros — APPROVED** [review]

**Pull request merged**
